prediction/src/algorithms/segment/src/levelset_segment.py

```diff
@@ -0,0 +1,46 @@
+import SimpleITK as sitk
+
+
+def segment_nodule(image: sitk.Image, nodule: dict, lower_threshold=-600, upper_threshold=500):
+    """
+
+    :param image: Input image
+    :param lower_threshold: Lower Hu threshold of the nodule to be segmented. Should be about the Hu of lung
+    :param upper_threshold: Upper Hu threshold of the nodule to be segmented. Should be less than the Hu of bone
+    :param nodule: dict containing the nodule coordinates
+    :return:
+    """
+
```

Comment:
Would you mind using [Google Style Docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)?

Reply:
Sorry. Oversight on my part. Fixed.